horace_core/sqw/PixelData/@MultipixBase/MultipixBase.m

```diff
@@ -0,0 +1,289 @@
+classdef MultipixBase < serializable
```

Comment:
Why `MultipixBase`? How does that correlate with the `pix(file|obj)_combine_info` which inherit it?